src/wizard/dotnet/xamarin.md

```diff
@@ -0,0 +1,74 @@
+---
+name: Xamarin
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/xamarin/
+support_level: alpha
+type: library
+---
+
+## Install the NuGet package
+
+```shell
+# Using Package Manager
+Install-Package Sentry.Xamarin.Forms -Version 1.0.0-alpha.3
```

Comment:
```suggestion
# For Xamarin.Forms
Install-Package Sentry.Xamarin.Forms -Version 1.0.0-alpha.3

# If you are not using Xamarin.Forms, but only Xamarin:
Install-Package Sentry.Xamarin -Version 1.0.0-alpha.3
```

And then also explain below what needs to be done if adding Xam.Forms (the integration thing)